src/dune_rules/odoc.ml

```diff
@@ -763,7 +767,7 @@ let out_files ctx (output : Output_format.t) odocs =
        (List.map odocs ~f:(fun odoc -> Path.build (out_file output odoc)))
 ;;
 
-let setup_lib_html_rules_def =
+let setup_lib_html_rules_def ~search_db =
```

Comment:
not for self, this might not be correct - search_db can change and it's not part of the memo input